src/backend/blockchain/src/routes.py

```diff
@@ -60,6 +109,7 @@ def proof():
 
     blockchain.add_to_wallet(miner_id, Blockchain.COINBASE_AMOUNT)
 
+    replicate('/addBlock', 'POST', data)
```

Comment:
This is actually a pretty clean solution, nicely done

Reply:
thank you!